src/library_async.js

```diff
@@ -79,46 +63,42 @@ mergeInto(LibraryManager.library, {
     instrumentWasmImports: function(imports) {
 #if ASYNCIFY_DEBUG
       err('asyncify instrumenting imports');
-#endif
-#if ASYNCIFY == 2
-      // TODO we could perhaps add an init function and put this there, but
-      //      this should work for now.
-      Asyncify.suspender = new WebAssembly.Suspender();
 #endif
       var ASYNCIFY_IMPORTS = {{{ JSON.stringify(ASYNCIFY_IMPORTS) }}}.map((x) => x.split('.')[1]);
       for (var x in imports) {
         (function(x) {
           var original = imports[x];
           var sig = original.sig;
           if (typeof original == 'function') {
-            var isAsyncifyImport = ASYNCIFY_IMPORTS.indexOf(x) >= 0 ||
-                                   x.startsWith('__asyncjs__');
 #if ASYNCIFY == 2
-            if (isAsyncifyImport) {
+            // Wrap all imports with a suspending web assembly function.
+            // TODO: wrap only async functions.
```

Comment:
is `isAsyncifyImport` that was deleted not the right code for that?

Reply:
It is, but for my first version of the JSPI binaryen pass it wraps all functions.

Reply:
Ok, sgtm to leave that for later.